packages/typespec-test/test/todo_non_branded/generated/typespec-ts/src/api/todoItems/attachments/index.ts

```diff
@@ -0,0 +1,93 @@
+// Licensed under the MIT License.
+
+import {
```

Comment:
I created a pr for this also, let me close my one then: https://github.com/Azure/autorest.typescript/pull/2911.